crates/forge_analyzer/src/definitions.rs

```diff
@@ -1695,17 +1703,20 @@ impl<'cx> FunctionAnalyzer<'cx> {
             }) => {
                 let cond = self.lower_expr(test, None);
                 let curr = self.block;
-                let rest = self.body.new_block();
-                let cons_block = self.body.new_block();
-                let alt_block = self.body.new_block();
+                let [temp1, temp2, temp3] = self.body.new_blocks();
+                let rest = self.body.new_blockbuilder();
+                let cons_block = self.body.new_blockbuilder();
+                let alt_block = self.body.new_blockbuilder();
                 self.set_curr_terminator(Terminator::If {
+                    // TODO: COME BACK TO?
```

Comment:
wdym come back to?

Reply:
Oh wait this was a note I made to myself while coding, thought I got rid of it in the latest commit but think I forgot to delete, will take out!